RocksDbSharp/RocksDbException.cs

```diff
@@ -1,13 +1,15 @@
 ﻿using System;
 using System.Runtime.InteropServices;
+using System.Text;
 
 namespace RocksDbSharp
 {
     public class RocksDbException : RocksDbSharpException
     {
         public RocksDbException(IntPtr errptr)
-            : base(Marshal.PtrToStringUni(errptr))
+            : base(Encoding.UTF8.GetString(Encoding.Unicode.GetBytes(Marshal.PtrToStringUni(errptr))))
         {
+            Marshal.PtrToStringUni(errptr);
```

Comment:
Why is this necessary?

Reply:
No, it is my mistake during debugging. I will remove it 😥